src/js/components/Form/Form.js

```diff
@@ -88,14 +89,20 @@ const Form = forwardRef(
           // re-run any validations, in case the validation
           // is checking across fields
           setErrors(prevErrors => {
-            const nextErrors = { ...prevErrors };
-            Object.keys(prevErrors).forEach(errName => {
-              if (validations.current[errName]) {
-                const nextError = validations.current[errName](data, nextValue);
-                updateErrors(nextErrors, errName, nextError);
-              }
-            });
-            return nextErrors;
+            if (prevErrors[name] && validations.current[name]) {
+              const nextErrors = { ...prevErrors };
+              Object.keys(prevErrors).forEach(errName => {
```

Comment:
I don't think we need to iterate over the errors, since we already know the name in question:
```
if (prev && current) {
  const next = { ...prev };
  updateErrors(next, name, validations.current[name](data, nextValue));
  return next;
}
```

Reply:
gotcha, made changes and tested.